solr/CHANGES.txt

```diff
@@ -41,6 +41,8 @@ New Features
 
 * SOLR-16409: Admin UI - Expose all highlighting parameters in the Query UI (Jeanie Lam via Eric Pugh)
 
+* SOLR-16438: Support optional split.setPreferredLeaders prop in shard split command. (Bruno Roustant)
```

Comment:
People debate this sort of thing but I think of this as an improvement to a feature and not a new feature.

Reply:
I don't see Solr Ref Guide changes